back-end/branch/models.py

```diff
@@ -0,0 +1,14 @@
+from django.db import models
+from basemodel.models import BaseModel
+from department.models import Department
+from degree.models import Degree
+
+# Create your models here
+
+class Branch(BaseModel):
+    name = models.TextField(blank=True,null=True,max_length=50)
```

Comment:
Use something like an enum here instead of a plain text field. Relevant documentation [here](https://docs.djangoproject.com/en/4.0/ref/models/fields/#field-choices-enum-types).

Reply:
We can use Enum but Institute might add new Branches. This may create issue

Reply:
So the plan is to use the Branch model as a field for placement data, right? In such a case, make sure that only admin has a write access to this model, and none of the services should expose this model outside.

Reply:
I didn't add access control. I will add once authentication is done.